docs/instrumentation-list.yaml

```
@@ -708,13 +732,27 @@ libraries:
       - org.jboss.logmanager:jboss-logmanager:[1.1.0.GA,)
   jdbc:
   - name: jdbc
+    description: |
+      Traces JDBC operations such as statements, batches, commits, and rollbacks by injecting byte‑code into JDBC drivers and common connection‑pool proxies. Each call produces a span named after the SQL verb, enriched with standard DB client attributes (system, database, operation, sanitized statement, peer address) and error details if an exception occurs.
```

```suggestion
      The JDBC instrumentation provides database client spans and metrics. Each call produces a span named after the SQL verb, enriched with standard DB client attributes (system, database, operation, sanitized statement, peer address) and error details if an exception occurs.
```
I removed some stuff that IMO isn't completely correct. We don't currently add commits or rollbacks and byte code stuff only applies to the agent.